src/App.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import { Provider } from 'react-redux';
+import { store } from './store';
+import { useAppSelector } from './store/hooks';
+import './App.css';
+import Modal from './componets/Modal';
+import MainPage from './componets/MainPage';
+import UncontrolledForm from './componets/forms/UncontrolledForm/UncontrolledForm';
+import ControlledForm from './componets/forms/ControlledForm/ControlledForm';
```

Comment:
Add extentsion to your IDE for checking misspelling `components`
./compone**n**ts/Modal'
<img width="592" height="60" alt="image" src="https://github.com/user-attachments/assets/41667b70-3327-4d07-a62d-8aeb74e986bf" />